lib/dal/src/migrations/U2302__actions.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE actions
```

Comment:
how will this fold into everything is a node or edge work? Do we need to make @nickgerace and @jhelwig aware?